internal/keycloakb/passwordgeneration.go

```diff
@@ -19,7 +21,9 @@ const (
 func appendCharacters(pwdElems []string, alphabet string, length int) []string {
 
 	for j := 0; j < length; j++ {
-		pwdElems = append(pwdElems, string(alphabet[rand.Intn(len(alphabet))]))
+		nBig, _ := crand.Int(crand.Reader, big.NewInt(int64(len(alphabet))))
```

Comment:
You could evaluate `big.NewInt(int64(len(alphabet)))` in a local variable `maxInt` outside of the loop to make it more readable (and very very little bit more performant)